apps/condo/domains/banking/access/BankTransaction.js

```diff
@@ -6,16 +6,26 @@ const { get } = require('lodash')
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 const { getById } = require('@open-condo/keystone/schema')
 
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
 const { checkPermissionInUserOrganizationOrRelatedOrganization, queryOrganizationEmployeeFor,
     queryOrganizationEmployeeFromRelatedOrganizationFor,
 } = require('@condo/domains/organization/utils/accessSchema')
+const { SERVICE } = require('@condo/domains/user/constants/common')
 
-async function canReadBankTransactions ({ authentication: { item: user } }) {
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
+async function canReadBankTransactions ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
     if (user.isAdmin) return {}
 
+    if (user.type === SERVICE) {
+        return {
+            integrationContext: { integration: { accessRights_some: { user: { id: user.id }, deletedAt: null } }, deletedAt: null  },
```

Comment:
```suggestion
            integrationContext: { integration: { accessRights_some: { user: { id: user.id, deletedAt: null }, deletedAt: null } }, deletedAt: null  },
```

Reply:
does not make sense, because if the user is deleted, the guard will work at the beginning and the request will be rejected without reaching this check